circleci/circleci-docs
# Use asset path helper for js asset includes

## Overview
For some reason we were not using the asset paths helper for a lot of our js assets, resulting in them not cache-busted when they were deployed. This fixes that for the js files in the default layout.

## How to Test
Build the site, load it. Open the network inspector, verify that the js files listed in the default layout are loading and have the hash included in their name.
